.clang-format

```diff
@@ -21,4 +21,5 @@ BreakBeforeBraces: Custom
 BraceWrapping:
   SplitEmptyFunction: false
   AfterCaseLabel: true
+QualifierAlignment: Left
```

Comment:
RIP, `const` on the right... I found that easier to read because its consistent behavior with its usage in the middle...

Reply:
Let me check the codebase again. If most of them left it is better to split PR for changing it. Anyway, lets invite more people so we can decide which is better @seladb @tigercosmos @clementperon. 

Note: We still have plenty of time because there are lots of warnings even for Common++ so it will take some time to fix PR :)

Reply:
I like the left one though.